code/modules/flufftext/Hallucination.dm

```diff
@@ -260,24 +256,24 @@ Gunshots/explosions/opening doors/less rare audio (done)
 		if(1) //Laser fight
 			for(var/i=0,i<hits,i++)
 				target << sound('sound/weapons/Laser.ogg',0,1,0,25)
-				sleep(rand(1,5))
+				sleep(rand(4,12))
 			target << sound(get_sfx("bodyfall"),0,1,0,25)
 		if(2) //Esword fight
 			target << sound('sound/weapons/saberon.ogg',0,1,0,15)
 			for(var/i=0,i<hits,i++)
 				target << sound('sound/weapons/blade1.ogg',,0,1,0,25)
-				sleep(rand(1,5))
+				sleep(rand(4,12))
```

Comment:
this should actually use the click cooldown defines; this is actually way too fast most of the time